app/shared/displacer/displacer.directive.ts

```diff
@@ -0,0 +1,11 @@
+import { Directive, TemplateRef, ViewContainerRef } from '@angular/core';
+import { TemplatePortal } from '@angular/material';
+
+@Directive({
+  selector: '[hipDisplacerPortal]'
+})
+export class DisplacerDirective extends TemplatePortal {
+  constructor(templateRef: TemplateRef<any>, viewContainerRef: ViewContainerRef) {
+    super(templateRef, viewContainerRef);
+  }
+}
```

Comment:
__[tslint]__
* file should end with a newline

<!-- Comment by [SideCI](https://sideci.com) -->